cppcache/src/Utils.cpp

```diff
@@ -173,20 +174,22 @@ const boost::dll::shared_library& getSharedLibrary(
 
   const auto& find = sharedLibraries.find(libraryName);
   if (find == sharedLibraries.end()) {
+    auto path = libraryName.empty() ? boost::dll::program_location()
```

Comment:
Great timing for me merging the boost::dll changes the other day. This change looks pretty straight forward!